src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -377,16 +380,16 @@ public virtual TagBuilder GenerateAntiForgery([NotNull] ViewContext viewContext)
             bool allowMultiple,
             object htmlAttributes)
         {
-            ICollection<string> ignored;
+            var currentValues = GetCurrentValues(viewContext, modelExplorer, expression, allowMultiple);
```

Comment:
current values of the expression -- whatever the method can find in ModelState, ViewData, the model, or the expression result.  "selected values" isn't quite correct because we aren't (yet) generating a `<select/>` or setting `SelectListItem.Selected`.